cmake/v_library.cmake

```diff
@@ -94,6 +94,10 @@ function(v_cc_library)
       ${V_DEFAULT_LINKOPTS}
       )
     target_compile_definitions(${_NAME} PUBLIC ${V_CC_LIB_DEFINES})
+    ## TODO: Remove this when fully switched over to OSSL
+    if (ENABLE_OSSL_IN_SEASTAR)
+      target_compile_definitions(${_NAME} PUBLIC ENABLE_OSSL_IN_SEASTAR)
```

Comment:
This should be attached to the seastar library and brought in by the dependency on it via `v::base`.

Or at least put it in `v::base`;

Reply:
yeah good point, there actually already is something there